charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -252,6 +260,9 @@ spec:
           volumeMounts:
           - name: consul-bin
             mountPath: /consul-bin
+          - mountPath: /consul/service
```

Comment:
We have used `consul-data` in some existing helm charts like mesh gateway and others and in other work like client token others, we have used `consul-login`. I will look at trying to align those outside of this PR.

Reply:
Thanks @jmurret !! I did realize that while I was working on the mesh-gateway and I should have fixed it up, but I ran into some issues during my testing and was addressing bugs one failure at a time and forgot to then come back and rename all the things! this would be really appreciated 🙏